flash/vision/classification/model.py

```diff
@@ -85,9 +83,12 @@ def __init__(
         if isinstance(backbone, tuple):
             self.backbone, num_features = backbone
         else:
-            self.backbone, num_features = backbone_and_num_features(backbone, pretrained=pretrained)
+            metadata = {"type": "resnet"} if "resnet" in backbone else {}
+            self.backbone, num_features = BACKBONES_REGISTRY.get(backbone,
+                                                                 **metadata)(pretrained=pretrained, **backbone_kwargs)
 
-        self.head = nn.Sequential(
+        head = head(num_features, num_classes) if isinstance(head, Callable) else head
```

Comment:
what else can be apart from Sequential ?

Reply:
users could want to define their own MLP output.

Reply:
Done ! Changed to FunctionType as nn.Module are callable